simulators/seissol/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Stage 1: Prepare the test environment
+FROM ubuntu:22.04 as test_env
+
+# Set frontend to be noninteractive to avoid prompts during installations
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install basic dependencies for test environment
+RUN apt-get update && apt-get install -y \
+    wget \
+    git \
+    unzip
+
+# Set up directories for examples and output
+RUN mkdir -p /home/seissol/examples /home/seissol/output
+
+# Clone SeisSol examples repository
+RUN git clone https://github.com/SeisSol/Examples.git /home/seissol/examples
+
+# Download specific test data for tpv33 example
+WORKDIR /home/seissol/examples/tpv33
+RUN wget https://zenodo.org/record/8042664/files/tpv33_half_sym.xdmf -O tpv33_half_sym.xdmf && \
+    wget https://zenodo.org/record/8042664/files/tpv33_half_sym -O tpv33_half_sym
+
+# Copy test simulation script into the container
+COPY ./test_sim.sh /home/seissol/examples/tpv33/test_sim.sh
+RUN chmod +x /home/seissol/examples/tpv33/test_sim.sh
+
+# Stage 2: Build the SeisSol environment
+FROM seissol/training:latest as build
```

Comment:
This is fine for now. But, as a note. In the future, we prefer to compile the simulator ourselves, this gives us more control on the binary locations, names, if we use mpi, openmp etc.

Reply:
okay you want us to build the docker image from start not from an existing docker image.
okay will work on that